portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,28 +27,20 @@
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
 
-  private List<String> datas;
-
-  @Override
-  public void init() {
-    datas = new ArrayList<>();
-    datas.add(
-        "I love coffee and my favorite chain is Philz!");
-    datas.add("My birthday is on July 19.");
-    datas.add("My favorite show is Game of Thrones.");
-    datas.add("I went to the Grace Hopper Conference last year!");
-    datas.add("My favorite class at CMU was 15-112 Fundamentals of Programming!");
-    datas.add("My favorite book is Catcher in the Rye.");
-    datas.add("One thing I want to learn more about is machine learning.");
-    datas.add("My ideal job is working on accessibility software for self-driving cars.");
-    datas.add("My favorite place at CMU to study is the Tepper Quad.");
-  }
+  private DataComment data;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String data = datas.get((int) (Math.random() * datas.size()));
+    DataComment object = new DataComment("mahima", "hello");
```

Comment:
'object' isn't a very descriptive name for a variable, something like 'comment' would be clearer